apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -460,6 +460,13 @@ const trackSiteEditorChangeTemplatePart = ( id ) => {
 	} );
 };
 
+const trackSiteEditorChangeContent = ( { type, slug } ) => {
+	tracksRecordEvent( 'wpcom_block_editor_nav_sidebar_item_edit', {
+		item_type: type,
+		item_slug: slug,
```

Comment:
It seems when we select a 'category' page we just get `item_slug` with the category name and no `item_type`.

Reply:
We don't have that information from the args. 😢  Used a selector to check the active menu. Type is determined based on that in this case.